manifests/cache_gems.pp

```diff
@@ -28,7 +28,7 @@
 
   package { 'builder':
     ensure   => present,
-    provider => 'gem',
+    provider => 'pe_gem',
```

Comment:
We were installing the gem in the wrong place.  This was like two weeks ago, so I don't remember all the details.